doc/api.odocl

```diff
@@ -1 +0,0 @@
-B64
```

Comment:
can you let an empty `doc` directory (for now) to make `topkg-jbuilder` happy (so that `topkg doc` would work).